runtime/tests/jsig/main.c

```diff
@@ -58,7 +58,15 @@
 
 #define PRIMARY_CALLED 1
 #define SECONDARY_CALLED 2
+#ifndef _AIX
 static volatile sig_atomic_t handlerCalled = 0;
+#else
```

Comment:
The coding standards ask the condition be documented like `#else /* !defined(_AIX) */`. Same for the #endif